apps/daimo-mobile/app.config.js

```diff
@@ -112,6 +112,12 @@ export default {
           "Allows Daimo to scan QR codes to pay other users or add new devices to your account.",
       },
     ],
+    [
+      "expo-contacts",
+      {
+        contactsPermission: "Allows Daimo to find and pay your friends.",
```

Comment:
should we say anything about local-only here?

Reply:
done